.github/workflows/cpu_ci.yml

```diff
@@ -42,5 +42,5 @@ jobs:
       - name: Type Checking
         uses: jakebailey/pyright-action@v1
 
-      - name: Run CPU Tests
-        run: pytest -m cpu
+      - name: Run normal tests, excluding GPU tests
+        run: pytest -m "not gpu"
```

Comment:
in a further PR i can do something like

```
    strategy:
      matrix:
        python-versions: [3.9, "3.10"]
        os: [ubuntu-18.04, macos-latest]
    runs-on: ${{ matrix.os }}
```
so that we don't have to keep remembering to edit these two lines